Makefile

```diff
@@ -85,10 +85,10 @@ lint-server: test-dependencies
 	flake8 elyra
 
 prettier-ui:
-	yarn prettier:check
+	yarn prettier
 
 eslint-ui:
-	yarn eslint:check --max-warnings=0
+	yarn eslint --max-warnings=0
```

Comment:
can you rebase? this should be fixed in master now

Reply:
done